src/libraries/Microsoft.PowerFx.Core/Public/Config/Features.cs

```diff
@@ -28,5 +28,10 @@ public enum Features : int
         /// Filter(A, A[@Value] = 2)
         /// </summary>
         DisableRowScopeDisambiguationSyntax = 0x4,
+
+        /// <summary>
+        /// All features enabled
+        /// </summary>
+        All = ~0
     }
```

Comment:
Adding this seems potentially risky. We don't want hosts to use this and pick up future flags unintentionally, I'd think

Reply:
This is for testing where I want to consider them all, even future ones, I'll add a comment on the risk you raise